docs/guides/migrating-from-1-to-2.md

```diff
@@ -0,0 +1,38 @@
+# React Dropdown Tree Select Migration Guide v1 to v2
+
+We've made some changes in the component between the versions which is covered in the following guide
+
+## Property changes
+
+| Description                                                              | Usage before                                                | Usage after                                                   |
+| ------------------------------------------------------------------------ | ----------------------------------------------------------- | ------------------------------------------------------------- |
+| Added a new radio select mode<br>Selection is done in a shared mode-prop | `simpleSelect={true}` / `simpleSelect`                      | `mode='simpleSelect'`                                         |
```

Comment:
We can release radio select before the other two. That way, people can still get the most out of v1 without upgrading to v2. radio select is backward compatible which is another reason I prefer it to be part of a v1 release. Thoughts?

Reply:
See [other pr:s comment](https://github.com/dowjones/react-dropdown-tree-select/pull/217#issuecomment-483514803). The same applies here.

Reply:
And now it's reverted